target/scripts/startup/setup-stack.sh

```diff
@@ -300,7 +300,7 @@ function _setup_dovecot_local_user
   local SLEEP_PERIOD='10'
   for (( COUNTER = 11 ; COUNTER >= 0 ; COUNTER-- ))
   do
-    if [[ $(grep -cE '.+@.+\|' /tmp/docker-mailserver/postfix-accounts.cf) -ge 1 ]]
+    if [[ $(grep -cE '.+@.+\|' /tmp/docker-mailserver/postfix-accounts.cf 2>dev/null || printf '0') -ge 1 ]]
```

Comment:
I would prefer a simple `echo` and avoid `printf` when not strictly needed:

```suggestion
    if [[ $(grep -cE '.+@.+\|' /tmp/docker-mailserver/postfix-accounts.cf 2>dev/null || echo '0') -ge 1 ]]
```

Or at least with a format string, to keep in sync with #2564:

```suggestion
    if [[ $(grep -cE '.+@.+\|' /tmp/docker-mailserver/postfix-accounts.cf 2>dev/null || printf '%s' '0') -ge 1 ]]
```

Reply:
I‘ll go with the format string:)